docs/proposals/dbaas/01-architecture/proposal.md

```diff
@@ -0,0 +1,695 @@
+{
+     "kubernetes_cluster_name": "lke75856"
+}
+'
+30.66
+
+TIME="%e\n" time curl -k -s -o file1.out --request POST --url https://localhost:8443/v1/management/DBaaS/DBClusters/List --header 'accept: application/json' --header 'authorization: Basic YWRtaW46YWRtaW4=' --header 'content-type: application/json' --data '
+{
+     "kubernetes_cluster_name": "lke75856"
+}
+'
+26.60
+
+TIME="%e\n" time curl -k -s --request POST --url https://localhost:8443/v1/management/DBaaS/DBClusters/List --header 'accept: application/json' --header 'authorization: Basic YWRtaW46YWRtaW4=' --header 'content-type: application/json' --data '
+{
+     "kubernetes_cluster_name": "lke75856"
+}
+'
+<html>
+<head><title>504 Gateway Time-out</title></head>
+<body>
+<center><h1>504 Gateway Time-out</h1></center>
+<hr><center>nginx</center>
+</body>
+</html>
+60.04
+```
+
+**PMM integrated with dbaas-operator and communicates via client-go library**
+
+```
+
+## Listing 10 clusters
+time curl -k -s -o file1.out --request POST --url https://localhost:8443/v1/management/DBaaS/DBClusters/List --header 'accept: application/json' --header 'authorization: Basic YWRtaW46YWRtaW4=' --header 'content-type: application/json' --data '{
+     "kubernetes_cluster_name": "minikube"
+}'
+
+0.38s
+
+time curl -k -s -o file1.out --request POST --url https://localhost:8443/v1/management/DBaaS/DBClusters/List --header 'accept: application/json' --header 'authorization: Basic YWRtaW46YWRtaW4=' --header 'content-type: application/json' --data '{
+     "kubernetes_cluster_name": "minikube"
+}'
+
+0.82s
+
+time curl -k -s -o file1.out --request POST --url https://localhost:8443/v1/management/DBaaS/DBClusters/List --header 'accept: application/json' --header 'authorization: Basic YWRtaW46YWRtaW4=' --header 'content-type: application/json' --data '{
+     "kubernetes_cluster_name": "minikube"
+}'
+
+0.30s
+
+## Listing 20 database clusters
+
+time curl -k -s -o file1.out --request POST --url https://localhost:8443/v1/management/DBaaS/DBClusters/List --header 'accept: application/json' --header 'authorization: Basic YWRtaW46YWRtaW4=' --header 'content-type: application/json' --data '{
+     "kubernetes_cluster_name": "minikube"
+}'
+0.67s
+
+time curl -k -s -o file1.out --request POST --url https://localhost:8443/v1/management/DBaaS/DBClusters/List --header 'accept: application/json' --header 'authorization: Basic YWRtaW46YWRtaW4=' --header 'content-type: application/json' --data '{
+     "kubernetes_cluster_name": "minikube"
+}'
+1.14s
+
+time curl -k -s -o file1.out --request POST --url https://localhost:8443/v1/management/DBaaS/DBClusters/List --header 'accept: application/json' --header 'authorization: Basic YWRtaW46YWRtaW4=' --header 'content-type: application/json' --data '{
+     "kubernetes_cluster_name": "minikube"
+}'
+0.76s
+
+
+
+```
+The environment for the benchmark above uses the same testing environment
```

Comment:
Sorry but I have doubts regarding equality of testing environments.
1. The 3rd test uses `"kubernetes_cluster_name": "minikube"` that points me to idea that K8S cluster is running on the same PC with PMM. So network latency and (that is more important) authentication in cloud provider perimeter (I suspect in the prev tests AWS has used) is absent. But exactly this 2 points may bring such big delays.
2. The comments in 3rd test say that there were 10 and 20 DB clusters inside K8S. But how many were in the 1rd and 2nd?